### PR TITLE
fix package.sh to properly start and stop Xvfb if necessary

### PR DESCRIPTION
**Pull request by mogoodrich**
package.sh didn't appear to be properly starting and stopping Xvfb. See comments inline.

### CODE REVIEW

File: ui/scripts/package.sh [hunk 1]
```diff
@@ -13,12 +13,13 @@ rm -rf $ROOT_DIR/target/${ZIP_FILE_NAME}*.zip
 npm install
 bower install
 
-pids=$(pgrep Xvfb)
```
Comment (mogoodrich) [inline ui/scripts/package.sh]: If Xvfb is not running, the exit code of the pgrep will be 1, and therefore the exit code of the variable assignment is also 1. Since "set -e" is set earlier in the commit, the script would always exit as this point.

File: ui/scripts/package.sh [hunk 2]
```diff
@@ -13,12 +13,13 @@ rm -rf $ROOT_DIR/target/${ZIP_FILE_NAME}*.zip
...
-if [ -n "$pids" ]; then
+if [ -z $(pgrep Xvfb) ]; then
```
Comment (mogoodrich) [inline ui/scripts/package.sh]: Putting the pgrep directly within the test prevents a non-zero exit code from prematurely terminating the script.

Also note that I believe we should be testing with -z here, not -n, since we want to return true if the value DOES have zero size.

File: ui/scripts/package.sh [hunk 3]
```diff
@@ -31,3 +32,9 @@ cd $ROOT_DIR/dist && zip -r ../target/${ZIP_FILE_NAME}_chrome.zip *
 cd ..
 grunt android
 cd $ROOT_DIR/dist && zip -r ../target/${ZIP_FILE_NAME}_android.zip *
+
+if [ -n $XVFB_PID ]; then
+    echo "Killing Xvfb process $XVFB_PID"
+    kill $XVFB_PID
```
Comment (mogoodrich) [inline ui/scripts/package.sh]: We also need to make sure we kill the Xvfb process if we started it--otherwise this script does not exit.

**Comment by sravanthi17** [comment]
@mogoodrich : Thanks for your contribution. We merged this commit and will be available from 0.83v

**Comment by mogoodrich** [comment]
Thanks!
